src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -265,9 +265,9 @@ private static bool OverloadResolutionResultIsValid<TMember>(ArrayBuilder<Member
             for (int i = 0; i < members.Count; i++)
             {
                 AddMemberToCandidateSet(
-                    members[i], results, members, typeArguments, receiver, arguments, completeResults,
-                    isMethodGroupConversion, allowRefOmittedArguments, containingTypeMapOpt, inferWithDynamic: inferWithDynamic,
-                    useSiteInfo: ref useSiteInfo, allowUnexpandedForm: allowUnexpandedForm);
+                    members[i], results, members, typeArguments, arguments, completeResults, isMethodGroupConversion,
```

Comment:
did an editor feature automatically move these arguments to the previous line?

Reply:
if we are making manual edits here, we might want to just do one-argument-per-line for this call.

Reply:
Yes, this was automatic. I did Change Signature and removed the unused parameter. There must be some heuristic in there for how to format ;-)
Sure, I can unwrap the arguments. It looks like the other call to this method uses one-argument-per-line.